tests/e2e/common/test_common.go

```diff
@@ -208,63 +211,62 @@ func (m *OCRv2TestState) initializeNodesInContractsMap() {
 // DeployContracts deploys contracts
 func (m *OCRv2TestState) DeployContracts(contractsDir string) {
 	m.NodeKeysBundle, m.err = CreateNodeKeysBundle(m.ChainlinkNodes)
-	Expect(m.err).ShouldNot(HaveOccurred())
+	require.NoError(m.T, m.err)
 	cd, err := solclient.NewContractDeployer(m.c, m.Env, nil)
-	Expect(err).ShouldNot(HaveOccurred())
+	require.NoError(m.T, err)
 	err = cd.LoadPrograms(contractsDir)
-	Expect(err).ShouldNot(HaveOccurred())
+	require.NoError(m.T, err)
 	err = cd.DeployAnchorProgramsRemote(contractsDir)
-	Expect(err).ShouldNot(HaveOccurred())
+	require.NoError(m.T, err)
 	cd.RegisterAnchorPrograms()
 	m.LinkToken, err = cd.DeployLinkTokenContract()
-	Expect(err).ShouldNot(HaveOccurred())
+	require.NoError(m.T, err)
 	err = FundOracles(m.c, m.NodeKeysBundle, big.NewFloat(1e4))
-	Expect(err).ShouldNot(HaveOccurred())
+	require.NoError(m.T, err)
 
 	m.initializeNodesInContractsMap()
 	g := errgroup.Group{}
 	for i := 0; i < len(m.ContractsNodeSetup); i++ {
 		i := i
 		g.Go(func() error {
-			defer ginkgo.GinkgoRecover()
 			cd, err := solclient.NewContractDeployer(m.c, m.Env, m.LinkToken)
-			Expect(err).ShouldNot(HaveOccurred())
+			require.NoError(m.T, err)
```

Comment:
Doesn't have to be done in this PR, but error messages would likely be helpful in some cases.